app/src/main/java/org/eyeseetea/malariacare/data/database/datasources/ConversionLocalDataSource.java

```diff
@@ -362,6 +372,9 @@ private void convertDataValues(ConvertFromSDKVisitor converter) {
             for (ProgramExtended program : ProgramExtended.getExtendedList(
                     SdkQueries.getProgramsForOrganisationUnit(organisationUnit.getId(),
                             ProgramType.WITHOUT_REGISTRATION))) {
+                if(!program.isValidProgram()){
+                    continue;
+                }
```

Comment:
Again, instead of adding this here, I would change the DB query to only retrieve the valid programs